release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3321,6 +3321,10 @@ Contact Red Hat Support for additional details and assistance.
 
 * There is a known issue in this release preventing access to the Web Terminal Operator after it has been installed. This issue will be fixed in a future {product-title} release. (link:https://issues.redhat.com/browse/OCPBUGS-14463[*OCPBUGS-14463*])
 
+* There is a known issue when upgrading the Cluster Network Operator from version 4.13 to 4.14.
+The conversion to the new OVN Kubernetes interconnect multi-zone architecture can lead to packet drops that cause network outages of up to 300ms per node. 
+The impact is on East/West traffic in local gateway mode with `routingViaHost` set to `true`.
```

Comment:
```suggestion
The impact is on East/West traffic in local gateway mode with `routingViaHost` set to `true`. (link:https://issues.redhat.com/browse/OCPBUGS-38891[*OCPBUGS-38891*])
```

Reply:
Thanks for the review @lahinson 
I've included the link as you suggested